lib/install/management/dispatcher.go

```diff
@@ -245,3 +254,47 @@ func (d *Dispatcher) CollectDiagnosticLogs() {
 		}
 	}
 }
+
+func (d *Dispatcher) DeleteVCH(conf *metadata.VirtualContainerHostConfigSpec) error {
+	log.Infof("Removing VMs")
+
+	if err := d.DeleteVCHInstances(conf); err != nil {
+		return err
+	}
+	if err := d.destroyResourcePoolIfEmpty(conf); err != nil {
+		return err
+	}
+	return d.removeNetwork(conf)
+}
+
+func (d *Dispatcher) DeleteVCHInstances(conf *metadata.VirtualContainerHostConfigSpec) error {
+	var errs []string
+
+	var err error
+	var vmm *vm.VirtualMachine
+	var children []*vm.VirtualMachine
+
+	if vmm, err = d.findAppliance(conf); err != nil {
+		return err
+	}
+	if vmm == nil {
```

Comment:
That means all VMs under the specified resource pool will be removed, even that resource pool is not created for VCH. Is that expected?